app/src/main/java/com/investrapp/investr/models/Cryptocurrency.java

```diff
@@ -0,0 +1,79 @@
+package com.investrapp.investr.models;
+
+import android.util.Log;
+
+import com.parse.ParseClassName;
+import com.parse.ParseObject;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+import okhttp3.Response;
+
+import static java.lang.System.in;
+
+/**
+ * Created by michaelsignorotti on 10/14/17.
+ */
+
+@ParseClassName("Cryptocurrency")
+public class Cryptocurrency extends ParseObject {
+
+    private String ticker;
```

Comment:
With these parse objects, you don't actually even need these class variables. Could probably just remove them.